examples/influxdb_data_model.ipynb

```diff
@@ -457,11 +458,8 @@
     "               The request was processed successfully\n",
     "             400:\n",
     "               Malformed syntax or bad query\n",
-    "\n",
-    "        Note\n",
-    "        ----\n",
-    "        `lsst.verify` measurement and InfluxDB measurement mean different things. \n",
```

Comment:
Why remove this? It's an important distinction.

Reply:
I agree, it is an important distinction - it just feels like the note is not in the right place, will find a better place for it.